DBAL/TracingConnectionFactory.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Auxmoney\OpentracingDoctrineDBALBundle\DBAL;
+
+use Auxmoney\OpentracingBundle\Service\Tracing;
+use Doctrine\Bundle\DoctrineBundle\ConnectionFactory as DoctrineConnectionFactory;
+use Doctrine\Common\EventManager;
+use Doctrine\DBAL\Configuration;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Types\Type;
+use ReflectionException;
+use ReflectionObject;
+
+final class TracingConnectionFactory
```

Comment:
Thats some nice showcase for composition over inheritance. But if you do not want to extend the doctrine ConnectionFactory, why not create your own interface for this class ?

Reply:
Because we do not actually use it at any given point. The container takes care of creating the factory. If Doctrine exposes an interface, we should adopt it asap.